Remotion/ObjectBinding/Web.Development.WebTesting/ControlObjects/BocDateTimeValueControlObject.cs

```diff
@@ -188,6 +188,11 @@ public IReadOnlyList<string> GetValidationErrors ()
       return GetDateValidationErrors();
     }
 
+    public IReadOnlyList<string> GetValidationErrorsForReadOnly ()
+    {
+      return GetValidationErrorsForReadOnly (GetLabeledElementScope());

```

Comment:
Is it okay to do this differently from z188?

Reply:
l.188 uses `GetDateScope()`, which uses the `input` scope, not available on a read-only control (being a `span`).

Reply:
OKay, the tests should take care of the rest, I'm assuming.